easybuild/easyblocks/t/trinity.py

```diff
@@ -279,7 +286,16 @@ def install_step(self):
             else:
                 raise EasyBuildError("Don't know how to set TRINITY_COMPILER for %s compiler", comp_fam)
 
-            cmd = "make TRINITY_COMPILER=%s" % trinity_compiler
+            explicit_make_args = ''
+            if LooseVersion(self.version) > "2" and LooseVersion(self.version) < "3":
+                explicit_make_args = 'all plugins'
+            # Delete CXXFLAGS CPPFLAGS and CFLAGS (interfere with plugins installation)
+            if LooseVersion(self.version) > "2" and LooseVersion(self.version) < "3":
+                del os.environ['CXXFLAGS']
+                del os.environ['CPPFLAGS']
+                del os.environ['CFLAGS']
```

Comment:
I rarely run into problems anymore when swapping GCC with the Intel compilers, I think BLAST+ is an exception here (do you have an issue for it?).
Usually, *if* is happens, it can be fixed by making sure 'strict' precision is used; the Intel compilers are (a lot) less strict w.r.t. floating-point precision that GCC is. EB already 'corrects' for that a bit by specifying certain precision-related flags by default when using the Intel compilers, but sometimes they need to be made a little bit stricter (by using `'strict': True` in `toolchainopts` in the easyconfig).
Also, I'm not sure that explicitly using GCC for the Trinity plugins is actually the intention; it looks like a case of "GCC is the only compiler" to me...

Also, this is relevant: https://github.com/hpcugent/easybuild-easyconfigs/blob/master/easybuild/easyconfigs/t/Trinity/trinity_ictce-no-jellyfish.patch .
Older versions of jellyfish seemed to fail to build with Intel compilers, this is clearly not the case anymore.